packages/frontend/src/features/catalog/components/CatalogMetadata.tsx

```diff
@@ -90,7 +91,7 @@ export const CatalogMetadata: FC = () => {
         }
     }, [metadataResults, selection, selectedFieldInTable]);
 
-    if (!metadata) return null;
+    if (!metadata) return <LoadingOverlay visible />;
```

Comment:
I can see this, only once, on the first metadata request, but not between metadata requests (on slow 3g network settings) , is this intended ? 

Perhaps we should `reset` the metadata when navigating, to force this overlay to appear between requests.

Reply:
I just tested this now https://github.com/lightdash/lightdash/pull/10265/commits/9d1ceba1ff7e9f4b165137227f6d9744e7cf6a5c and with a slow/fast 3g it will display the overlay. I recommend we try this on Analytics